CONTRIBUTING.md

```diff
@@ -168,6 +179,23 @@ Someone might be able to warn you in advance that your change will have wide imp
 However, this can only happen if you discuss your proposed changes early and often.
 It's often better to check *before* contributing that you're setting off on the right path.
 
+## Fixing style guide inconsistencies
+
+Fantomas tries to keep up with the style guides, but as these are living documents, it can occur that something is listed in the style that Fantomas is not respecting.
+In this case, please create an issue using our [online tool](https://fsprojects.github.io/fantomas-tools/#/).  
+Copy the code snippet from the guide and add a link to the section of the guide that is not being respected.  
+The maintainers will then add the `bug (stylistic)` to confirm the bug may be fixed in Fantomas. In most cases, it may seem obvious that the case can be fixed.
```

Comment:
```suggestion
The maintainers will then add the `bug (stylistic)` to confirm the bug is fixable in Fantomas. In most cases, it may seem obvious that the case can be fixed.
```

("may be fixed" could mean "might already be fixed")